bundles/CoreBundle/src/Command/Bundle/ListCommand.php

```diff
@@ -15,132 +15,141 @@
 namespace Pimcore\Bundle\CoreBundle\Command\Bundle;
 
 use const JSON_PRETTY_PRINT;
+use JsonException;
 use Pimcore\Extension\Bundle\Exception\BundleNotFoundException;
-use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use function json_encode;
+use function Symfony\Component\String\s;
 
 /**
  * @internal
  */
+#[AsCommand(
+    name: 'pimcore:bundle:list',
+    description: 'Lists all pimcore bundles and their enabled/installed state'
+)]
 class ListCommand extends AbstractBundleCommand
 {
     protected function configure(): void
     {
-        $this
-            ->setName($this->buildName('list'))
-            ->setDescription('Lists all pimcore bundles and their enabled/installed state')
-            ->addOption(
-                'fully-qualified-classnames',
-                'f',
-                InputOption::VALUE_NONE,
-                'Show fully qualified class names instead of short names'
-            )
-            ->addOption(
-                'details',
-                'd',
-                InputOption::VALUE_NONE,
-                'Show more details of pimcore bundles e.g. version, description etc.'
-            )
-            ->addOption('json', null, InputOption::VALUE_NONE, 'Return data as JSON')
-        ;
+        $this->addOption(
+            'fully-qualified-classnames',
+            'f',
+            InputOption::VALUE_NONE,
+            'Show fully qualified class names instead of short names'
+        )->addOption(
+            'details',
+            'd',
+            InputOption::VALUE_NONE,
+            'Show more details of pimcore bundles e.g. version, description etc.'
+        )->addOption(
+            'json',
+            null,
+            InputOption::VALUE_NONE,
+            'Return data as JSON'
+        );
     }
 
+    /**
+     * @throws JsonException
+     */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $details = $input->getOption('details');
-        $returnData = [
-            'headers' => [
-                'Bundle',
-                'Enabled',
-                'Installed',
-                $input->hasOption('json') ? 'Installable' : 'I?',
-                $input->hasOption('json') ? 'Uninstallable' : 'UI?',
-                'Priority',
-            ],
-            'rows' => [],
+        $headers = [
+            'Bundle',
+            'Enabled',
+            'Installed',
+            'Installable',
+            'Uninstallable',
```

Comment:
```suggestion
            $input->getOption('json') ? 'Installable' : 'I?',
            $input->getOption('json') ? 'Uninstallable' : 'UI?',
```

I feel there it was meant to have a narrower column but the `hasOption` was not used as intended, it's always truthy as if it checks if the option is an available one, by replacing with getOption would appear as

<img width="1037" height="335" alt="image" src="https://github.com/user-attachments/assets/1d3e6aff-75da-4869-9255-2564f2e87e6b" />